xtext/org.icyphy.linguafranca/src/org/icyphy/ASTUtils.xtend

```diff
@@ -1167,8 +1174,95 @@ class ASTUtils {
             return new TimeValue(0, TimeUnit.NONE)
         }
     }
+        
+    /**
+     * If the parameter has a literal integer value, then return that value.
+     * Otherwise, return -1.
+     * @param parameter The parameter.
+     * @return The parameter's integer value or -1
+     *  if the value cannot be determined or it is not an integer.
+     */
+    def static int intValue(Parameter parameter) {
```

Comment:
Maybe call this `widthValue` or `portWidth` to make clear that this attempts to retrieve a _positive_ integer and returns -1 if it fails to?

Reply:
This function is no longer used. I have removed it.